model/src/main/proto/profile.proto

```diff
@@ -41,6 +41,15 @@ message Profile {
 
   // Represents the time the user's profile was created.
   int64 date_created_timestamp_ms = 9;
+
+  // Represents user selected story-text-size.
+  float story_text_size = 10;
```

Comment:
Isn't the story text size actually a multiplier? I thought this was an in-app version of the system-level setting for font size, or is this something else?

Reply:
This contains proto now.